mlrun/data_types/to_pandas.py

```diff
@@ -178,7 +178,7 @@ def toPandas(spark_df):
         if isinstance(field.dataType, IntegralType) and pandas_col.isnull().any():
             dtype[fieldIdx] = np.float64
         if isinstance(field.dataType, BooleanType) and pandas_col.isnull().any():
-            dtype[fieldIdx] = np.object
+            dtype[fieldIdx] = np.object_
```

Comment:
The recommendation is to replace this deprecated type with a plain `object`, not with the internal type.

```suggestion
            dtype[fieldIdx] = object
```

Reply:
Same goes for  np.float64. Lets be consistent

Reply:
Please see https://numpy.org/devdocs/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated:
> For float and complex you can use float64 and complex128 if you wish to be more explicit about the precision.

Reply:
But feel free to change it to `float` if you think that makes more sense.

Reply:
np.float64 is in may places - I don't want to change it. So I don't agree with you, but I don't feel it is important enough. Changing it as per your suggestion